modules/apps/product-navigation/product-navigation-product-menu-web/src/main/resources/META-INF/resources/portlet/init.jsp

```diff
@@ -50,4 +54,9 @@ page import="java.util.Objects" %>
 
 <%
 ProductMenuDisplayContext productMenuDisplayContext = new ProductMenuDisplayContext(liferayPortletRequest, liferayPortletResponse);
+
+long guestGroupId = GroupLocalServiceUtil.getGroup(themeDisplay.getCompanyId(), GroupConstants.GUEST).getGroupId();
```

Comment:
Hey @ChrisKian , @jesseyeh-liferay , @wanderlast 
Guest is not ever the default site, you can modified it via properties, so, it is better to use PropsValues.VIRTUAL_HOSTS_DEFAULT_SITE_NAME instead the constant GUEST.
Best regards,